spec/controllers/admin/organizations_controller_spec.rb

```diff
@@ -3,7 +3,7 @@
 describe Admin::OrganizationsController do
   let!(:admin) { create(:admin) }
   let!(:organization) { create(:organization) }
-  let!(:user) { create(:user) }
+  let!(:user) { create(:user, email: 'user@osem.io') }
```

Comment:
What's the reason for giving the user a static email, instead of one generated by the factory? 
In your tests, you can just use `user.email` instead of `'user@osem.io'`.